packages/lib-classifier/src/store/ClassificationStore.js

```diff
@@ -159,6 +160,14 @@ const ClassificationStore = types
         if (process.browser) {
           console.log('Completed classification', classificationToSubmit)
         }
+
+        if (self.demoMode) {
+          // subject advance is observing for this to know when to advance the queue
+          self.loadingState = asyncStates.posting
+          if (process.browser) console.log('Demo mode enabled. No classification submitted.')
+          return Promise.resolve(true)
+        }
```

Comment:
Sorry, so the loading state isn't being set in two places. It's only being set once.

Reply:
I tried this early on in the previous version of this feature's implementation and I tried again just now to remind myself why I didn't do this. I get MST errors about trying to read or write to an object that is no longer part of a state tree with the classification meaning it's not advancing the subject queue and subsequently resetting the classification correctly.  Perhaps if it's broken out into more separate actions, it would work correctly, but that's more refactoring than I think is in scope of this PR.